### ISSUE TITLE
Improvement request: ability to change the unordered list bullet character to use

### ISSUE THREAD

**Issue by dstj** [mysticmind/reversemarkdown-net#70]
Hi,

Just found out about this Converter and I'm quite pleased so far!

The system to which I'm pushing the converted Markdown only seems to accepts `*` as unordered list (`<ol><li>`) bullet lead in character, not the `-` used.

So, I'd like to suggest an improvement to add a setting to pick the bullet character to use. It could be `-` by default, but others are equally valid it appears. The [markdown guide](https://www.markdownguide.org/basic-syntax#lists-1) says:

> Unordered Lists
> To create an unordered list, add dashes (-), asterisks (*), or plus signs (+) in front of line items. Indent one or more items to create a nested list.

That would resolve the issue with my "non-fully-markdown-compliant" other system...

I'm guessing in would only require modifying [this line of code](https://github.com/mysticmind/reversemarkdown-net/blob/911561aaed1bf59cb65c980581ce4402906bafde/src/ReverseMarkdown/Converters/Li.cs#L45) right here:

```c#
        private string PrefixFor(HtmlNode node)
        {
            if (node.ParentNode != null && node.ParentNode.Name == "ol")
            {
                // index are zero based hence add one
                var index = node.ParentNode.SelectNodes("./li").IndexOf(node) + 1;
                return $"{index}. ";
            }
            else
            {
                return "- ";
            }
        }
```

**Comment by dstj** [comment]
I created PR #71 for it... was simple enough.

**Comment by mysticmind** [comment]
@dstj thanks for the PR, will check and merge the same during day tomorrow.

**Comment by mysticmind** [comment]
Closed via #71
